bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Position"),
+        SKOS.broader: QUDTQK.Dimensionless,
+        QUDT.hasDimensionVector: QUDTDV["A0E0L0I0M0H0T0D1"],
+    },
+    "Power_Factor": {OWL.sameAs: QUDTQK["PowerFactor"]},
     "Pressure": {
-        "subclasses": {
-            "Atmospheric_Pressure": {},
-            "Static_Pressure": {},
-            "Velocity_Pressure": {},
+        OWL.sameAs: QUDTQK["Pressure"],
+        "subconcepts": {
+            "Atmospheric_Pressure": {OWL.sameAs: QUDTQK["AtmosphericPressure"]},
+            "Dynamic_Pressure": {},
+            "Static_Pressure": {OWL.sameAs: QUDTQK["StaticPressure"]},
+            "Velocity_Pressure": {
+                OWL.sameAs: [QUDTQK["DynamicPressure"], BRICK["Dynamic_Pressure"]],
+            },
+        },
+    },
+    "Radiance": {
+        OWL.sameAs: QUDTQK["Radiance"],
+        "subconcepts": {
+            "Solar_Radiance": {
+                QUDT.applicableUnit: [UNIT["W-PER-M2-SR"]],
+                SKOS.definition: Literal(
+                    "The amount of light that passes through or is emitted from the sun and falls within a given solid angle in a specified direction",
+                ),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Solar_Radiance"),
+                SKOS.broader: QUDTQK.Radiance,
+            }
+        },
+    },
+    "Speed": {
+        # TODO: fan speed is not meter/sec
+        OWL.sameAs: QUDTQK["Speed"],
+        "subconcepts": {
+            "Wind_Speed": {
+                QUDT.applicableUnit: [
+                    UNIT["M-PER-HR"],
+                    UNIT["KiloM-PER-HR"],
+                    UNIT["FT-PER-HR"],
+                    UNIT["MI-PER-HR"],
+                    UNIT["M-PER-SEC"],
+                    UNIT["KiloM-PER-SEC"],
+                    UNIT["FT-PER-SEC"],
+                    UNIT["MI-PER-SEC"],
+                ],
+                SKOS.definition: Literal(
+                    "Measured speed of wind, caused by air moving from high to low pressure",
+                ),
+                QUDT.hasDimensionVector: QUDTDV["A0E0L1I0M0H0T-1D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Wind_Speed"),
+                SKOS.broader: QUDTQK.Speed,
+            }
         },
     },
-    "Radiance": {"subclasses": {"Solar_Radiance": {}}},
-    "Speed": {"subclasses": {"Wind_Speed": {}}},
     "Temperature": {
-        "subclasses": {
-            "Operative_Temperature": {},
-            "Radiant_Temperature": {},
-            "Dry_Bulb_Temperature": {},
-            "Wet_Bulb_Temperature": {},
+        OWL.sameAs: QUDTQK["ThermodynamicTemperature"],
+        "subconcepts": {
+            "Operative_Temperature": {
+                QUDT.applicableUnit: [UNIT["DEG_F"], UNIT["DEG_C"], UNIT["K"]],
```

Comment:
Can we use more generic methodology for this like Operative_Temperature's applicableUnits are same as Temperature's? (Not sure if it's true for this though.)

Reply:
What do you mean by a more generic methodology? Operative Temperature needs its own definition and is a more specific concept than just regular Temperature